src/styles/BookList.css

```diff
@@ -25,12 +27,13 @@
 }
 
 .list-item {
+  align-items: center;
+  background-color: #fff;
+  box-shadow: 0 0 15px 10px rgba(238, 239, 243, 0.8);
   display: flex;
-  justify-content: space-between;
-  padding: 40px;
-  background-color: white;
-  border: 0.5px solid rgb(210, 181, 181);
-  margin: 11px 0;
+  gap: 10%;
```

Comment:
- [ ] Kindly use the correct font family which is `RobotoSlab` for this section. 🤝️
- [ ] Please put the texts `CURRENT CHAPTER` and `UPDATE PROGRESS` on a single line as per the design requirements. 🥸️
- [ ] Kindly check the color you are using for the text `CURRENT CHAPTER` 🙂️

| Your design                                                             | Zeplin design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/224475900-cbdb5506-b920-4c1f-84d5-020f206b960d.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/224475897-d8a4d603-f736-457b-9c43-836036245c27.png" width="300">|